### PR TITLE
rr_install

### PR DESCRIPTION
**Pull request by robertzk**

### CODE REVIEW

**Comment by robertzk** [comment]
@peterhurford

File: rrzsh.plugin.zsh [hunk 1]
```diff
@@ -23,6 +24,13 @@ rr_test() {
   fi
 }
 
+rr_install() {
+  shift
+  if [ $# -eq 0 ]; then; Rscript -e "library(methods); library(devtools); install()";
```
Comment (peterhurford) [inline rrzsh.plugin.zsh:29]: Why `install()` with no arguments?

File: rrzsh.plugin.zsh [hunk 2]
```diff
@@ -23,6 +24,13 @@ rr_test() {
...
+  else; Rscript -e "library(methods); library(devtools); test($1);"
```
Comment (peterhurford) [inline rrzsh.plugin.zsh]: Why `test()`?

File: rrzsh.plugin.zsh (hunk 1 above)
Comment (robertzk) [inline rrzsh.plugin.zsh:29]: `test` is with no arguments as well, `$# -eq 0`

File: rrzsh.plugin.zsh (hunk 2 above)
Comment (robertzk) [inline rrzsh.plugin.zsh]: Oops!

**Comment by robertzk** [comment]
Fixed, how about now? @peterhurford

**Pull request merged**